app/client/ui2018/theme.ts

```diff
@@ -60,8 +59,8 @@ let _gristThemeObs: Computed<Theme> | undefined;
 export function gristThemeObs() {
   if (!_gristThemeObs) {
     _gristThemeObs = Computed.create(null, (use) => {
-      // Custom CSS is incompatible with custom themes.
-      if (getGristConfig().enableCustomCss) { return DEFAULT_LIGHT_THEME; }
+      // Default to light theme if themes are disabled in config.
+      if (!getGristConfig().features?.includes('themes')) { return DEFAULT_LIGHT_THEME; }
```

Comment:
There's a `isFeatureEnabled` helper in `gristUrls.ts` that you can use here and elsewhere.
```suggestion
      if (!isFeatureEnabled('themes')) { return DEFAULT_LIGHT_THEME; }
```

Reply:
Oh thanks!